src/target/target.cc

```diff
@@ -210,7 +210,14 @@ ObjectRef TargetInternal::ParseType(const std::string& str,
     // Parsing integer
     int v;
     if (!(is >> v)) {
-      throw Error(": Cannot parse into type \"Integer\" from string: " + str);
+      // Bool is a subclass of IntImm, so allow textual boolean values.
```

Comment:
It would be better to use `to_lower` to canonicalize the string.

Reply:
Done.